projects/libreoffice/fonts.conf

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<fontconfig>
+  <dir>/out/instdir/share/fonts/truetype</dir>
```

Comment:
Actually, I take that back. @mikea, we flatten the $OUT directory when zipping right?